playbook-website/app/views/pages/code_snippets/position_token.txt

```diff
@@ -1,5 +1,5 @@
-@import "tokens/positioning
+@import "token/positioning"
 
 .selector {
-    z-index: $z_10;
+     position: $absolute
```

Comment:
```suggestion
     position: $absolute;
```

Reply:
Hey @carloslimasd , everything looks great! Just a tiny edit, missing the ending semi colon here. you don't have to do a commit, just click 'commit suggestion'